internal/core/model.rs

```diff
@@ -1146,13 +1146,16 @@ pub struct TableColumn {
     pub title: SharedString,
 
     /// The minimum column width
-    pub min_width: f64,
+    pub min_width: Coord,
 
     /// The horizontal column stretch
     pub horizontal_stretch: f32,
 
     /// Sorts the column
     pub sort_order: SortOrder,
+
+    /// the actual width of the column
+    pub width: Coord,
```

Comment:
Since this data structure is public API, I think that it would make sense to document explicitly whether the width and min_width here is logical pixels or physical pixels.

Reply:
added a commit with docs.